requirements-dev.txt

```diff
@@ -10,4 +10,4 @@ git+https://github.com/yaml/pyyaml.git
 git+https://github.com/python-jsonschema/jsonschema
 
 scipy>=0.0.dev0
-numpy>=0.0.dev0
+numpy>=0.0.dev0 ; python_version > "3.8"
```

Comment:
Can you put an upper pin on numpy for Python 3.8 in the pyproject.toml

Reply:
That's a great suggestion. However, I'd rather have a public change to the requirements in a separate PR with a corresponding changelog entry. If you'd rather have those here I'll make the changes.

Reply:
I think we can just retitle this PR and add a changelog. That way all the numpy/Python 3.8 stuff is all organized together.